Task_1/setup.py

```diff
@@ -28,11 +28,11 @@
     ],
     include_package_data=True,
     install_requires=[
-        'openfl @ git+https://github.com/intel/openfl.git@f4b28d710e2be31cdfa7487fdb4e8cb3a1387a5f',
-        'GANDLF @ git+https://github.com/CBICA/GaNDLF.git@e4d0d4bfdf4076130817001a98dfb90189956278',
+        'openfl @ git+https://github.com/securefederatedai/openfl.git@kta-intel/fets-2024-patch-1',
+        'GANDLF @ git+https://github.com/CBICA/GaNDLF.git@0.1.0',
```

Comment:
```suggestion
        'GANDLF @ git+https://github.com/mlcommons/GaNDLF.git@0.1.0',
```

Can we open an issue where we add a CI run on both GaNDLF and OpenFL master/develop branches respectively?

Reply:
This is still unresolved.